nilearn/reporting/_get_clusters_table.py

```diff
@@ -125,7 +165,19 @@ def _pare_subpeaks(xyz, ijk, vals, min_distance):
 
 
 def _get_val(row, input_arr):
-    """Small function for extracting values from array based on index.
+    """Extract values from array based on index.
+
+    Parameters
+    ----------
+    row : :obj:`tuple` of length 3
+        3-length index into ``input_arr``.
+    input_arr : 3D :obj:`numpy.ndarray`
+        Array from which to extract value.
+
+    Returns
+    -------
+    scalar
```

Comment:
Maybe you could add the return type for consistency here.

Reply:
```suggestion
    :obj:`float` or :obj:`int`
```